services/app-api/utils/types/reports.ts

```diff
@@ -63,6 +64,14 @@ export interface ModalDrawerReportPageShape extends ReportPageShapeBase {
   form?: never;
 }
 
+export interface EntityOverlayPageShape extends ReportPageShapeBase {
+  entityType: string;
+  form: FormJson;
+  dashboard?: never;
+  modalForm: never;
+  drawerForm?: never;
```

Comment:
```suggestion
  modalForm: never;
```
Is it possible since this shape isn't going to use it to just remove the call? I legitimately don't know. Just would be nice. Totally understandable if typescript doesn't like this.